rllib/algorithms/callbacks.py

```diff
@@ -427,12 +428,14 @@ def on_episode_end(
         env_index: Optional[int] = None,
         **kwargs,
     ) -> None:
+        gc.collect()
         snapshot = tracemalloc.take_snapshot()
         top_stats = snapshot.statistics("lineno")
 
         for stat in top_stats[:10]:
             count = stat.count
-            size = stat.size
+            # Convert total size from Bytes to KiB.
+            size = stat.size / 1024
```

Comment:
@sven1977 Can confirm that size in measured in KiB here.
https://docs.python.org/3/library/tracemalloc.html#get-the-traceback-of-a-memory-block